src/styles/block/cards.scss

```diff
@@ -0,0 +1,90 @@
+.card__link,
+.card__product,
+.card__stars,
+.card__button {
+  text-decoration: none;
+  color: $prim-color;
+  display: inline-block;
+}
```

Comment:
Don't group selectors in block like that. If you similar styles - use mixin or extend.